api/main.py

```
@@ -54,87 +88,157 @@ async def root():
-        file: CSV file containing transaction data
-        
-    Returns:
-        JSON with credit_score, risk_probability, risk_category, and interpretation
-    """
+# -----------------------------------------------------
+# Prediction Endpoint
+# -----------------------------------------------------
+
+@app.post("/api/predict", response_model=PredictionResponse)
+async def predict(
+    file: UploadFile = File(...),
```

Ruff (0.14.4)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)